docs/utils/index.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/prefer-default-export */
+import SunCalc from 'suncalc';
+import isWithinInterval from 'date-fns/isWithinInterval';
+import { theme } from '../../constants';
+
+export const getAutoTheme = () => {
+  const currentDate = new Date();
+  let currentLatitude = '';
+  let currentLongitude = '';
+
+  navigator.geolocation.getCurrentPosition(
```

Comment:
I'm not sure about getting sun position, because user needs to grant access to geolocation in the browser.

Reply:
Can we use something simpler? Maybe just check if the current time is greater or lower than threshold?